packages/docusaurus-theme-search-algolia/src/theme/SearchBar/index.tsx

```diff
@@ -107,9 +101,10 @@ function DocSearch({
       configFacetFilters;
 
   // We let user override default searchParameters if she wants to
-  const searchParameters: DocSearchProps['searchParameters'] = {
+  const searchParameters: DocSearchModalProps['searchParameters'] = {
     ...props.searchParameters,
     facetFilters,
+    indexName: props.indexName,
```

Comment:
Doesn't look dangerous but wondering what's the impact of adding this `indexName` here considering we already pass the `indexName` prop (spread) to the docsearch modal component?